x-pack/plugins/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -157,6 +158,9 @@ export const ruleRegistrySearchStrategyProvider = (
               );
             });
           }
+
+          response.inspect = { dsl: [JSON.stringify(params)] };
```

Comment:
I think we should wrap the `JSON.stringify` with a try/catch, it can throw error.